### PR TITLE
Update README links, rand generation in workers, consistent CLI calls, remove flaky worker_create script

### PR DESCRIPTION
**Pull request by alastairdglennie**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,6 +81,12 @@ show how sources generated from the schema could be linked in the worker
 binary. See `schema/CMakeLists.txt` which creates a library with all generated
 sources.
 
+You can see (and edit) the content of the snapshot in text format by running a command to convert it:
+
+```
+spatial project history snapshot convert --input <path> --input-format binary --output <path> --output-format text
```
Comment (improbable-nickkrempel) [inline README.md]: Do you want to use consistent `--foo=bar` style here too?

File: README.md [hunk 2]
```diff
@@ -101,10 +108,10 @@ As usual set the `project_name` field in `spatialos.json` to match your SpatialO
 
 ```
 spatial cloud upload <AssemblyName>
-spatial cloud launch <AssemblyName> default_launch.json <deploymentname>
+spatial cloud launch <AssemblyName> default_launch.json <deploymentname> --snapshot=<snapshotpath>
```
Comment (improbable-nickkrempel) [inline README.md]: Inconsistent capitalisation (`<AssemblyName>` vs <deploymentname>`).

File: workers/External/src/startup.cc [hunk 3]
```diff
@@ -65,7 +65,8 @@ std::string get_random_characters(size_t count) {
 
 // Entry point
 int main(int argc, char** argv) {
-    srand(time(nullptr));
+    auto now = std::chrono::high_resolution_clock::now();
+    srand(std::chrono::time_point_cast<std::chrono::nanoseconds>(now).time_since_epoch().count());
```
Comment (improbable-nickkrempel) [inline workers/External/src/startup.cc]: One day we can go fully C++ and use `<random>` too :)

Comment (improbable-nickkrempel) [inline workers/External/src/startup.cc]: (Also, where's the `#include` for `rand` and `srand`? Probably want `#include <cstdlib>` and use `std::srand`, `std::rand`.)

File: README.md (hunk 1 above)
Comment (alastairdglennie) [reply README.md]: Changing the style yep. Would opt to leave out the quotes around <path> for the sake of clarity.

**Pull request merged**